standard/src/compression/mod.rs

```diff
@@ -1,3 +1,203 @@
+//! Implementations for many widely used compression algorithms, including
+//! [DEFLATE](crate::compression::deflate), [lz4](crate::compression::lz4),
+//! [zstd](crate::compression::zstd) and [brotli](crate::compression::brotli).
```

Comment:
Can be addressed later: may be nice to explain the tradeoffs with the different compression algorithms

Reply:
Good point, I may need to do some documentation enhancements next time for the standard crate.